src/oapi/flexible_gpus.rs

```diff
@@ -53,6 +53,14 @@ impl Input {
     }
 
     pub fn fill_resource_flexible_gpus(&self, resources: &mut Resources) {
+        if self.flexible_gpus.is_empty() && self.need_default_resource {
+            resources.resources.push(Resource::FlexibleGpu(FlexibleGpu {
+                account_id: self.account_id(),
+                read_date_rfc3339: self.fetch_date.map(|date| date.to_rfc3339()),
+                region: self.region.clone(),
+                ..Default::default()
+            }));
```

Comment:
I guess you should return here, because if flexible_gpu is empty, all the loop bellow will be skip